apps/remote_control/lib/lexical/remote_control/search/store.ex

```diff
@@ -28,12 +29,19 @@ defmodule Lexical.RemoteControl.Search.Store do
              {:ok, new_entries, paths_to_delete} | {:error, term()})
 
   @backend Application.compile_env(:remote_control, :search_store_backend, Store.Backends.Ets)
+  @flush_interval_ms Application.compile_env(
+                       :remote_control,
+                       :search_store_quiescent_period_ms,
+                       2500
+                     )
+
+  defrecordp :store, state: nil, update_ref: nil
 
   use GenServer
   require Logger
 
   def stop do
-    GenServer.call(__MODULE__, :drop)
+    GenServer.call(__MODULE__, :on_stop)
```

Comment:
Would it be better to use the `terminate/2` callback here? According to the docs, `GenServer.stop/1` will already execute the terminate callback, and it also allows us to potentially do this cleanup when the store is killed by other means.